### PR TITLE
 Port enet.fs to C#(enet.cs) and add proxy and fix a small mistake

### PR DESCRIPTION
**Pull request by SurmanPP**

### CODE REVIEW

File: scripts/enet.fs
Comment (flp5) [inline scripts/enet.fs:1]: just delete the file: in case it is needed for whatever reason, just use git

File: README.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ Further notes on the usage and troubleshooting of every script can be found in t
 
 1. [dc.py](scripts/dc/dc.py)
    produces `{mapcode}-{gamemodecode}.pcapng` for each of the 140 - 2 bugged combinations of map-gamemode - files consiting of packets sent between a client and a local server over a 30 second period of the client connecting, spawning and disconnecting.  
-2. [enet.fs](scripts/enet.fs)
-   mimics a basic client or server and collects and pretty-prints the packets received from the official server or the official client respectively.  
\ No newline at end of file
+2. [enet.cs](scripts/enet.cs)(superseds enet.fs)
```
Comment (flp5) [inline README.md]: no need for the (supersedes enet.fs) part

**Review by flp5 — CHANGES_REQUESTED** [review]
change this and lgtm

**Comment by SurmanPP** [comment]
Changed the commit 👍🏻

**Pull request merged**